set.mm

```diff
@@ -625406,7 +625406,7 @@ family of sets (implicit).  (Contributed by Stefan O'Rear,
        over functions and points in N-dimensional space (which are equivalent
        to functions from an "index set").  Many of the following theorems exist
        to transfer standard facts about functions to elements of function
-       sets._ (Contributed by Stefan O'Rear, 30-Aug-2014.)  (Revised by Stefan
+       sets_.  (Contributed by Stefan O'Rear, 30-Aug-2014.)  (Revised by Stefan
```

Comment:
would this work? there is a period in the sentence before

Reply:
the ending underscore has to have a letter before it and no letter after it, so yes this should work

Reply:
Before:

![image](https://github.com/metamath/set.mm/assets/16878/20f4b3c9-c168-488c-b474-1c88b099a735)

After:

![image](https://github.com/metamath/set.mm/assets/16878/d86d2707-79b5-4d91-a44b-0b982eb101c8)